sirepo_flyer.py

```diff
@@ -0,0 +1,241 @@
+import datetime
+import hashlib
+import os
+import time as ttime
+from collections import deque
+from multiprocessing import Process
+from pathlib import Path
+
+from ophyd.sim import NullStatus, new_uid
+
+from sirepo_bluesky import SirepoBluesky
+from srw_handler import read_srw_file
+
+from re_config import *
```

Comment:
Let's move this line in the `if __name__ == '__main__'` block, and comment it out.